roles/kustomize_deploy/defaults/main.yml

```diff
@@ -41,98 +43,157 @@ cifmw_kustomize_deploy_architecture_repo_version: "HEAD"
+cifmw_kustomize_deploy_kustomizations_dest_dir: >-
   {{
     [
       cifmw_kustomize_deploy_destfiles_basedir,
       'artifacts',
-      'ci_gen_kustomize_values',
-      'network-values',
-      'values.yaml'
+      'kustomize_deploy'
     ] | path_join
   }}
 
-cifmw_kustomize_deploy_cp_source_files: >-
+cifmw_kustomize_deploy_check_mode: false
+cifmw_kustomize_deploy_keep_generated_crs: true
+
+# operators resources
+
+cifmw_kustomize_deploy_olm_source_files: >-
   {{
     [
       cifmw_kustomize_deploy_architecture_repo_dest_dir,
-      cifmw_kustomize_deploy_architecture_examples_va_path,
-      cifmw_architecture_va_scenario
+      cifmw_kustomize_deploy_architecture_examples_common_path,
+      'olm'
     ] | path_join
   }}
 
-cifmw_kustomize_deploy_cp_values_dest_file: >-
+cifmw_kustomize_deploy_olm_dest_file: >-
   {{
     [
-      cifmw_kustomize_deploy_cp_source_files,
-      'values.yml'
+      cifmw_kustomize_deploy_kustomizations_dest_dir,
+      'olm.yaml'
     ] | path_join
   }}
 
-cifmw_kustomize_deploy_kustomizations_dest_dir: >-
+cifmw_kustomize_deploy_metallb_source_files: >-
   {{
     [
-      cifmw_kustomize_deploy_destfiles_basedir,
-      'artifacts',
-      'kustomize_deploy'
+      cifmw_kustomize_deploy_architecture_repo_dest_dir,
+      cifmw_kustomize_deploy_architecture_examples_common_path,
+      'metallb'
     ] | path_join
   }}
 
-cifmw_kustomize_deploy_olm_dest_file: >-
+cifmw_kustomize_deploy_metallb_dest_file: >-
   {{
     [
       cifmw_kustomize_deploy_kustomizations_dest_dir,
-      'olm.yml'
+      'metallb.yaml'
     ] | path_join
   }}
 
-cifmw_kustomize_deploy_olm_source_files: >-
+cifmw_kustomize_deploy_nmstate_source_files: >-
   {{
     [
       cifmw_kustomize_deploy_architecture_repo_dest_dir,
       cifmw_kustomize_deploy_architecture_examples_common_path,
-      'olm'
+      'nmstate'
     ] | path_join
   }}
 
-cifmw_kustomize_deploy_metallb_dest_file: >-
+cifmw_kustomize_deploy_nmstate_dest_file: >-
   {{
     [
       cifmw_kustomize_deploy_kustomizations_dest_dir,
-      'metallb.yml'
+      'nmstate.yaml'
     ] | path_join
   }}
 
-cifmw_kustomize_deploy_metallb_source_files: >-
+# controlsplane resources
+
+cifmw_kustomize_deploy_nncp_source_files: >-
   {{
     [
-      cifmw_kustomize_deploy_architecture_repo_dest_dir,
-      cifmw_kustomize_deploy_architecture_examples_common_path,
-      'metallb'
+      cifmw_kustomize_deploy_cp_source_files,
+      'nncp'
     ] | path_join
   }}
 
-cifmw_kustomize_deploy_nmstate_dest_file: >-
+cifmw_kustomize_deploy_nncp_values_dest_file: >-
+  {{
+    [
+      cifmw_kustomize_deploy_nncp_source_files,
+      'values.yaml'
+    ] | path_join
+  }}
+
+cifmw_kustomize_deploy_nncp_dest_file: >-
   {{
     [
       cifmw_kustomize_deploy_kustomizations_dest_dir,
-      'nmstate.yml'
+      'nncp.yaml'
     ] | path_join
   }}
 
-cifmw_kustomize_deploy_nmstate_source_files: >-
+cifmw_kustomize_deploy_cp_source_files: >-
```

Comment:
Should we need to add 'control-plane' into the end of the path with PR 91?

Reply:
mmh, probably, yes.

(Probably i'll make this pr as draft again and wait until PR 91 get merged to avoid re-re-re-working on it)